analysers/analyser_merge_parking_FR_BNLS.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights Vincent Bergeot (thanks Frédéric Rodrigo) 2020             ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from .Analyser_Merge import Analyser_Merge, Source, CSV, Load, Mapping, Select, Generate
+
+
+class Analyser_Merge_Parking_FR_BNLS(Analyser_Merge):
+    def __init__(self, config, logger = None):
+        Analyser_Merge.__init__(self, config, logger)
+
+        doc_main = dict(
+            trap = T_(
+'''It is not because it is from and Opdata source that it is good data. Review it before integrating. Your are an OSM contributors, not a machin to do blind import.''')
```

Comment:
```suggestion
'''It is not because it is from and open data source that it is good data. Review it before integrating. Your are an OSM contributors, not a machin to do blind import.''')
```